### PR TITLE
Move isotovideo to OpenQA::Isotovideo::Runner

### PR DESCRIPTION
**Pull request by perlpunk**
This moves code from isotovideo and OpenQA::Isotovideo::CommandHandler to OpenQA::Isotovideo::Runner, by small steps.

It was thoroughly tested on some dedicated workers.

Issues: https://progress.opensuse.org/issues/81899 https://progress.opensuse.org/issues/121045

### CODE REVIEW

**Review by okurz — CHANGES_REQUESTED** [review]
+1

needs test coverage for OpenQA/Isotovideo/Runner.pm

**Comment by ggardet** [comment]
I tested it on a local machine which showed the issue previously. First run failed, but next 7 succeeded.
Later today, I plan to deploy this PR on some aarch64 workers on o3 to test it live on next Tumbleweed snapshot.

**Comment by kalikiana** [comment]
@perlpunk Going by @ggardet's [comment on the ticket](https://progress.opensuse.org/issues/121045#note-25) perhaps we can go ahead with this branch 🤩

File: xt/00-tidy.t [hunk 1]
```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/perl
 # Copyright 2021 SUSE LLC
 # SPDX-License-Identifier: GPL-2.0-or-later
-
 use Test::Most;
+pass "no thanks";
+done_testing;
+__DATA__
+
```
Comment (okurz) [inline xt/00-tidy.t:8]: needs to be reverted

File: isotovideo [hunk 2]
```diff
@@ -129,160 +114,41 @@ my $return_code = 1;
...
-stop_commands('test execution ended');
+$runner->stop_commands('test execution ended');
 
-if ($testfd) {
+if ($runner->testfd) {
     $return_code = 1;    # unusual shutdown
-    CORE::close $testfd;
-    stop_autotest;
+    CORE::close $runner->testfd;
+    $runner->stop_autotest();
```
Comment (kalikiana) [inline isotovideo:151]: I guess these two lines should also have `# uncoverable statement` markers

**Comment by kalikiana** [comment]
> This is a proof of concept, so maybe things can be made nicer, but if we were abe to test this on all platforms, I think it would be a good start.

@perlpunk Please adjust the description. It's not a PoC at this point.

File: xt/00-tidy.t (hunk 1 above)
Comment (perlpunk) [reply xt/00-tidy.t]: oops, done

File: isotovideo (hunk 2 above)
Comment (perlpunk) [reply isotovideo]: done

**Comment by perlpunk** [comment]
> @perlpunk Please adjust the description. It's not a PoC at this point.

done

**Review by kalikiana — APPROVED** [review]
Awesome sauce!

**Review by okurz — APPROVED** [review]

**Pull request merged**